modules/ravestate_nlp/triple.py

```diff
@@ -69,10 +69,12 @@ def to_tuple(self) -> Tuple[str, str, str]:
     def match_either_lemma(self, pred: Optional[Set[str]] = None, subj: Optional[Set[str]] = None, obj: Optional[Set[str]] = None):
         if pred and (self._predicate.lemma_ in pred or self._predicate_aux.lemma_ in pred):
             return True
-        if subj and self._subject.lemma_ in subj:
-            return True
-        if obj and self._object.lemma_ in obj:
-            return True
+        if subj:
+            if self._subject.lemma_ in subj or any(child.lemma_ in subj for child in self._subject.children):
```

Comment:
This is going to crash when `_subject` or `_object` is a QuestionWord. Just add en empty `children` member to the `QuestionWord` class.